cpr_data/cpr_zooplankton_map.sql

```diff
@@ -0,0 +1,25 @@
+-- This view is the basis for the WMS layer (seen on step 2 on AODN Portal).
+-- It also provides the metadata columns for all the zooplankton products.
+CREATE MATERIALIZED VIEW cpr_zooplankton_map AS
+  SELECT
+    s.trip_code AS "TripCode",
+    s.region AS "Region",
+    s.latitude AS "Latitude",
+    s.longitude AS "Longitude",
+    s.sampledateutc AT TIME ZONE 'UTC' AS "SampleDate_UTC",
```

Comment:
Don't need `AT TIME ZONE 'UTC'` - same as in [`bgc_trip_metadata`](https://github.com/aodn/public-schema/blob/c6e65f83fbbb7f64a88b78f5582b87dc28c04676/bgc_data/bgc_trip_metadata.sql#L10)